snek-code.c

```diff
@@ -1177,6 +1179,11 @@ snek_code_run(snek_code_t *code_in)
 					snek_id_del(id);
 				}
 				break;
+			case snek_op_assert:
+				if (!snek_poly_true(snek_a)) {
+					snek_error_0("AssertionError");
+				}
```

Comment:
Python:
>>> assert True
>>>
Snek:
> assert True
1
>

This can be easily fixed by setting snek_a to SNEK_NULL just after the if() statement here.